src/app/controllers/authentication.server.controller.js

```diff
@@ -100,9 +109,25 @@ exports.verifyPermission = (requiredPermission) =>
    * @param {function} next - Next middleware function
    */
   (req, res, next) => {
-    // Admins always have sufficient permission
-    let hasSufficientPermission =
+    const isFormAdmin =
       String(req.form.admin.id) === String(req.session.user._id)
+
+    // Forbidden if requiredPersmission is admin but user is not
+    if (!isFormAdmin && requiredPermission === PERMISSIONS.ADMIN) {
+      logger.error(
+        `User ${req.session.uer} not authorized to perform ${requiredPermission} operation on Form: ${req.form.title}.`,
```

Comment:
should be `req.session.user` instead of `req.session.uer`

Reply:
`User ${req.session.user} not authorized to perform ${requiredPermission} operation on Form ${req.form._id} with title: ${req.form.title}.` Easier to search through logs if we log the `formId`

Reply:
Good catch. Thanks!